proxy/ProxySession.h

```diff
@@ -181,3 +181,89 @@ class ProxySession : public VConnection
   // aborts.
   bool m_active = false;
 };
+
+///////////////////
+// INLINE
+
+static inline int64_t next_cs_id = 0;
+
+inline int64_t
+ProxySession::next_connection_id()
+{
+  return ink_atomic_increment(&next_cs_id, 1);
+}
```

Comment:
Does more than one function use (the name) next_cs_id?  If not, why not
```
inline int64_t
ProxySession::next_connection_id()
{
  static int64_t next_cs_id = 0;
  return ink_atomic_increment(&next_cs_id, 1);
}
```
Pushing things to the lowest scope possible make future maintenance easier.

Reply:
I don't think that is what he wants to do on this PR. You can make a PR for it.